config/advanced-install/namespaced-controller-wo-crds.yaml

```diff
@@ -136,158 +136,79 @@ metadata:
-          natsImage: nats:2.8.1
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.8.1-alpine
-          natsImage: nats:2.8.1-alpine
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: nats-server
-        - version: 2.8.3
-          natsImage: nats:2.8.3
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.8.3-alpine
-          natsImage: nats:2.8.3-alpine
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: nats-server
-        - version: 2.9.0
-          natsImage: nats:2.9.0
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.9.0-alpine
-          natsImage: nats:2.9.0-alpine
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: nats-server
-        - version: 2.9.6
-          natsImage: nats:2.9.6
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.9.8
-          natsImage: nats:2.9.8
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.9.15
-          natsImage: nats:2.9.15
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.10.3
-          natsImage: nats:2.10.3
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.10.11
-          natsImage: nats:2.10.11
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
-        - version: 2.10.17
-          natsImage: nats:2.10.17
-          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
-          startCommand: /nats-server
+  controller-config.yaml: "# Within a cluster, setting \"instance\" can be used to
```

Comment:
use `|` and make it readable and editable?

Reply:
This is ruined by `kustomize` build when multiple lines of comments are in place. 

Updated to one line comments made it work.